pkgs/tools/misc/zoxide/default.nix

```
@@ -10,13 +10,13 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "zoxide";
-  version = "unstable-2023-11-20";
+  version = "0.9.3";
 
   src = fetchFromGitHub {
     owner = "ajeetdsouza";
     repo = "zoxide";
-    rev = "3022cf3686b85288e6fbecb2bd23ad113fd83f3b";
-    sha256 = "sha256-ut+/F7cQ5Xamb7T45a78i0mjqnNG9/73jPNaDLxzAx8=";
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-dxdBXoV9X2oA5ig6PK/GuNLRRzO1E0zHJwVFQp7GY2g=";
```

```suggestion
    hash = "sha256-dxdBXoV9X2oA5ig6PK/GuNLRRzO1E0zHJwVFQp7GY2g=";
```
I'd like to modernize this.

---

You can also use `cargoHash` instead of `cargoSha256` 🦀

---

```
─❯ nix-prefetch-github --rev v0.9.3 ajeetdsouza zoxide
{
    "owner": "ajeetdsouza",
    "repo": "zoxide",
    "rev": "cbb8e77d60c6a8b5599d37f1732de41894afd7f3",
    "hash": "sha256-dxdBXoV9X2oA5ig6PK/GuNLRRzO1E0zHJwVFQp7GY2g="
}
```